### PR TITLE
Remove unused features, and simplify configuration structure

### PR DESCRIPTION
**Pull request by yann-eugone**
Because it is unclear how the environment feature works, it has been dropped

Because the configuration structure is easier to understand/evolve, it has been flattened

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,64 +36,40 @@ composer require yokai/safe-command-bundle
...
+    Yokai\SafeCommandBundle\YokaiSafeCommandBundle::class => ['prod' => true],
 ];
 ```
 
+> [!NOTE]
+> The bundle is enabled only for `prod` here, but you are free to do whatever you want.
+
 ### Configuration
 
+The bundle comes with few some commands disabled by default (from Symfony's standards), but
```
Comment (J-Ben87) [inline README.md]: `few` or `some`? ^^

File: README.md [hunk 2]
```diff
@@ -36,64 +36,40 @@ composer require yokai/safe-command-bundle
...
-    environments:
-        enabled: false
-        environments:
-            - prod
+when@prod:
+    yokai_safe_command:
+        standard: []
+```
+
+you add your own commands:
```
Comment (J-Ben87) [inline README.md]:
```suggestion
or add your own commands:
```

File: README.md [hunk 3]
```diff
@@ -36,64 +36,40 @@ composer require yokai/safe-command-bundle
...
+        custom:
+          - 'vendor:my:dev-command'
+          - 'app:my:dev-command'
+```
+
+> [!NOTE]
+> "standard" disabled commands are viewable via the command:
+> ```
+> bin/console config:dump-reference yokai_safe_command
+> ```
```
Comment (J-Ben87) [inline README.md:76]: You may add a note mentioning that standard and custom commands are merged?

**Review by J-Ben87 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yann-eugone) [reply README.md]: some :smile:

File: README.md (hunk 2 above)
Comment (yann-eugone) [reply README.md]: It was connected to the "but" few lines above, and it was kind of a list in my mind...
I changed the structure (for the best I hope)

**Review by J-Ben87 — APPROVED** [review]

**Pull request merged**